src/utilities/lib/visibility.scss

```diff
@@ -0,0 +1,45 @@
+$_tbds-visibility-types: (
+  "visible",
+  "hidden",
+  "collapse",
+) !default;
+
+@mixin _tbds-visibility-at-only {
+  border: 0;
+  clip: rect(0 0 0 0);
+  height: auto;
+  margin: 0;
+  overflow: hidden;
+  padding: 0;
+  position: absolute;
+  white-space: nowrap;
+  width: 1px;
+}
+
+@each $visibility-type in $_tbds-visibility-types {
+  @include tbds-generate-breakpoint-classes(
+    $selector: "tbds-visibility-" + $visibility-type,
+    $property: "visibility",
+    $value: $visibility-type,
+    $important: true,
+  );
+}
+
+@include tbds-generate-breakpoint-classes(
+  $selector: "tbds-visibility-none",
+  $property: "display",
+  $value: "none",
+  $important: true,
+);
+
+.tbds-visibility-at-only {
+  @include _tbds-visibility-at-only();
+}
+
+@each $breakpoint, $breakpoint-value in $tbds-breakpoints {
+  @media (min-width: $breakpoint-value) {
+    .tbds-visibility-at-only\@#{$breakpoint} {
+      @include _tbds-visibility-at-only();
+    }
+  }
+}
```

Comment:
I don’t think I’ve seen a need yet for the utility classes generated in this partial. I don’t want to lose this work, but how would you feel about focusing this PR on closing #43 and adding these utilities in the future when the need arises? 

It can be quite easy to expand the system rapidly, but it requires ongoing maintenance. Given our small team and that we are not in a position to work on this every day, I’ve been keen to keep it lean and expand only when required. There is perhaps a documentation update needed somewhere to make this more clear?

Reply:
I hear you loud and clear. I'm going to update this to pair the scope way, way down, and then maybe try and capture some of the learnings in the CONTRIBUTING.

Thanks for the thorough review here, I really appreciate you taking the time.